torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -828,11 +828,15 @@ void AliasDb::analyzeImpl(Node* node) {
 
     // Do sanity checks on the alias annotation
     TORCH_INTERNAL_ASSERT(
-        formal->containedTypes().size() == 0,
+        formal->containedTypes().size() <= 1,
         "Composite types for alias analysis not yet supported");
     TORCH_INTERNAL_ASSERT(
         !formal->isWildcardBefore(),
         "Doesn't make sense for a input value to begin as a wildcard");
+    if (formal->containedTypes().size() == 1 && formal->beforeSets().empty()) {
```

Comment:
oh ok, IIUC, this is because `formal->containedTypes().size() == 1 && formal->beforeSets().empty()` is still a valid state that technically means "no alias annotations". Maybe that's worth a comment?